pkg/minikube/machine/start.go

```diff
@@ -313,6 +314,23 @@ func postStartSetup(h *host.Host, mc config.ClusterConfig) error {
 		return nil
 	}
 
+	if driver.IsNone(h.DriverName) {
```

Comment:
`cri-dockerd` & `dockerd` are only required if the `docker` container-runtime is being used, but currently it's being checked for all container-runtimes.

Reply:
Added check for docker container-runtime:

```
// If none driver with docker container-runtime, require cri-dockerd and dockerd.
	if driver.IsNone(h.DriverName) && mc.KubernetesConfig.ContainerRuntime == constants.Docker {
...
```